.github/workflows/playwright-actions.yml

```diff
@@ -229,7 +229,7 @@ jobs:
         run: OPTIONS_REPOSITORY_IMPORT_FILTER=small make repos-import
 
       - name: Run nightly job to trigger snapshot for redhat repos
-        run: go run cmd/external-repos/main.go process-repos
+        run: go run cmd/external-repos/main.go snapshot https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2/os
```

Comment:
I think you are missing a slash after the URL, if I try doing this locally on a clean setup after importing the small RH repo, it doesn't schedule any tasks, with the slash it does 😅

Reply:
thanks!